src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
```

Comment:
Move all repeatable colors to CSS variables and move them to :root selector.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)